dist/index.js

```diff
@@ -0,0 +1,82 @@
+class BookStore {
+  constructor() {
+    this.books = [];
+  }
+
+  addBook(title, author) {
+    const book = {
+      title: title,
+      author: author
+    };
```

Comment:
- [OPTIONAL] I highly suggest that give the `book` object an `id` property that will make it unique, when you're trying to remove ir